rotkehlchen/db/settings.py

```diff
@@ -351,6 +359,11 @@ def serialize_db_setting(
         value = value.serialize()  # pylint: disable=no-member
     elif setting == 'address_name_priority' and is_modifiable is True:
         value = json.dumps(value)
+    elif setting == 'evmchains_to_skip_detection':
+        if is_modifiable is True:
+            value = json.dumps([x.to_name() for x in value])
+        else:
+            value = [x.to_name() for x in value]
```

Comment:
maybe is better to just do 

```suggestion
        value = [x.to_name() for x in value]
        if is_modifiable is True:
            value = json.dumps(value)
```

Reply:
maybe later since all green now